### PR TITLE
Bump to latest version

### PR DESCRIPTION
**Pull request by pgherveou**

### CODE REVIEW

File: node/src/chain_spec/dev.rs
Comment (pgherveou) [inline node/src/chain_spec/dev.rs:7]:
```suggestion
```

File: node/src/command.rs
Comment (pgherveou) [inline node/src/command.rs:228]:
```suggestion
```

**Review by ascjones — APPROVED** [review]

**Comment by ascjones** [comment]
CI failing though

**Review by sergejparity — APPROVED** [review]

**Review by smiasojed — APPROVED** [review]

**Comment by pgherveou** [comment]
I am going to wait before merging this, 
The build requires for now to hardcode this environment variable.
Looks like the problem might be solved on master and in upcoming crate.io release

https://github.com/paritytech/substrate-contracts-node/blob/be83efe93f018862b12b0f71d550de712babd03a/.gitlab-ci.yml#L23-L26

**Comment by ascjones** [comment]
> Looks like the problem might be solved on master and in upcoming crate.io release

Has this been done? I see there is e.g. a new version of https://crates.io/crates/pallet-contracts/27.0.0 released 10 days ago

**Comment by pgherveou** [comment]
> > Looks like the problem might be solved on master and in upcoming crate.io release

Yes it's solved on master not on the crates.io packages.
We can merge this PR if we are fine with giving up cargo releases
I haven't worked on updating the doc regarding it, nor created a process to use crates when they are released

File: .gitlab-ci.yml
Comment (pgherveou) [inline .gitlab-ci.yml:93]:
```suggestion
    - rustup component add rust-src wasm32-unknown-unknown
```

**Pull request merged**
